<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>changeConfigDialog</class>
 <widget class="QDialog" name="changeConfigDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>472</width>
    <height>283</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Commit</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item alignment="Qt::AlignHCenter">
    <widget class="QLabel" name="changeListLabel">
     <property name="font">
      <font>
       <family>Verdana</family>
      </font>
     </property>
     <property name="text">
      <string>Change List</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTextEdit" name="changeListText">
     <property name="font">
      <font>
       <family>Verdana</family>
      </font>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="buttonsHL">
     <item>
      <widget class="QPushButton" name="undoButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <family>Verdana</family>
        </font>
       </property>
       <property name="text">
        <string>Undo</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="commitButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <family>Verdana</family>
        </font>
       </property>
       <property name="text">
        <string>Commit</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
